content/plugin/sdkv2/testing/acceptance-tests/index.mdx

```diff
@@ -85,43 +85,142 @@ func TestAccXxx(*testing.T)
 
 ## Running Acceptance Tests
 
-A `terraform` binary of version 0.12.26 or above must be available on the local system for the acceptance test driver to run, either on the system `$PATH` or supplied via the `TF_ACC_TERRAFORM_PATH` environment variable described below. If you are running acceptance tests in a CI environment, we recommend that you download and install Terraform with a prior build step.
+~> **NOTE**: Acceptance tests typically create and destroy actual infrastructure resources, possibly incurring expenses during or after the test duration, and often require network connectivity and provider-specific credentials.
```

Comment:
I also don't love starting out with a note. Could we maybe move this to be just under a description of what acceptance tests do?

```suggestion
~> **NOTE**: Acceptance tests typically create and destroy actual infrastructure resources, possibly incurring expenses during or after the test. They also often require network connectivity and provider-specific credentials.
```